homeassistant/components/sensor/tellduslive.py

```diff
@@ -87,9 +90,7 @@ def _value_as_humidity(self):
     @property
     def name(self):
         """Return the name of the sensor."""
-        return '{} {}'.format(
-            super().name,
-            self.quantity_name or '')
+        return '{} {}'.format(super().name, self.quantity_name or '')
```

Comment:
Will this leave a space after the first word in the name if quantity name is empty?

Reply:
Probably yes, but this is just a reformat edit. I'll make inte nicer though.

Reply:
Strip is good!

Reply:
Which do you prefer? 
The first second one seems to be more efficient but also longer and harder to read.

```python
return '{} {}'.format(super().name, self.quantity_name or '').strip()
return '{} {}'.format(super().name, self.quantity_name) if self.quantity_name else super().name
```

Reply:
I like the top one with strip.